tool/travis.sh

```diff
@@ -4,7 +4,7 @@
 set -e
 
 function clean {
```

Comment:
@jmesserly I think you should be able to kill the `clean` function entirely from the travis script. Could you try?